kronofoto/archive/templates/archive/photo_partial.html

```diff
@@ -49,10 +43,26 @@
     {% endblock %}
 </div>
 
+<div hx-swap-oob="innerHTML:#add-to-list-popup .photo-menu-popup__wrapper">
+    Loading...
+    <script>
+      $('#add-to-list-popup').attr('hx-get', '{% krono_url "kronofoto:popup-add-to-list" photo=photo.id %}');
+      window.htmx.process($('#add-to-list-popup').get(0))
```

Comment:
We need a better way to get the htmx instance. window.htmx will not be reliable with more than web component on a page, or if we have a web component preview page.